packs/data/classfeatures.db/energy-barrier.json

```diff
@@ -0,0 +1,53 @@
+{
+    "_id": "xndlv9T3JgYYUtf8",
```

Comment:
Pipeline is failing because of an ID collision between this and Metallic Reactance. Make sure to make a new item in Foundry then edit it and export it to compendium to avoid ID collisions.